pysat/_params.py

```diff
@@ -219,10 +221,12 @@ def __str__(self, long_str=True):
         return out_str
 
     def __getitem__(self, item):
+        """Get item from Parameters."""
         return self.data[item]
 
     def __setitem__(self, key, value):
-        # Update current settings
+        """Update current settings in Parameters."""
+
```

Comment:
Does python ignore docstrings for __setitem__ like it does with the setter on properties?  If so, this info may need to be put in __getitem__.

Reply:
Not sure 🦈

Reply:
@rstoneback, is the above construction correct?

Reply:
The docstrings are correct. If we want them in the API docs though then the special members option needs to be invoked. I'm ok with it as is.

Reply:
testing out locally, adding the :special-members: in the api file works.  I can push that here so it's taken care of.

Reply:
or conversely, we can make it a new issue to add throughout docs where it makes sense.

Reply:
I'm leaning more towards adding it where it makes sense.